frontend/public/components/RBAC/bindings.jsx

```diff
@@ -296,6 +296,7 @@ export const RoleBindingsPage = ({
 }) => (
   <MultiListPage
     canCreate={true}
+    createAccessReview={{ model: modelFor('RoleBinding'), namespace }}
```

Comment:
This RBAC check is wrong. The form lets you create either a ClusterRoleBinding or a RoleBinding and lets you select the namespace. This check assumes it's a RoleBinding in a specific namespace.

Reply:
Changed to do a FLAGS.SHOW_OPENSHIFT_START_GUIDE check